net_handler/tcp_conn.c

```diff
@@ -181,11 +183,10 @@ static int recv_new_msg (int conn_fd, int challenge_fd)
 		}
 		text__free_unpacked(inputs, NULL);
 
-		log_printf(DEBUG, "entering CHALLENGE mode. running coding challenges!");
+		log_printf(INFO, "entering CHALLENGE mode. running coding challenges!");
```

Comment:
I don't think any of these statements should be INFO. I feel like we should almost never send logs from runtime to the student unless its an error.

Reply:
I feel like it would be useful to know, since like suppose a student uploads some code and runs it and presses "Autonomous" on Dawn. If they get nothing on the console, the student might think that something is broken with runtime or the connection is bad; if we do send something (entering <mode> mode) and then it still prints nothing then they know that the system is working fine and it's their code that's breaking.